ironfish/src/primitives/blockheader.ts

```diff
@@ -292,3 +205,86 @@ export class BlockHeaderSerde {
     )
   }
 }
+
+export function isBlockLater(a: BlockHeader, b: BlockHeader): boolean {
```

Comment:
Just moving these functions to below so that `BlockHeader` class can be at the top of the `blockheader.ts` file